Documentation/doc/Documentation/Developer_manual/Chapter_code_format.txt

```diff
@@ -243,13 +243,13 @@ The first list of items are meant as rules, <I>i.e.</I>, you should follow them.
   modify the object to which it is applied,
   <I>e.g.</I>, `class A { int f( void) const; };`. This should also be
   done when it is only conceptually `const`.
-  This means that the member function `f()` is `c`onst as seen from
+  This means that the member function `f()` is `const` as seen from
   the outside, but internally it may modify some data members
-  that are declared `m`utable. An example
+  that are declared `mutable`. An example
   is the caching of results from expensive computations.  For more
   information about conceptually `c`onst functions and mutable data
```

Comment:
was the fix to replace \`c\`onst with \`const\`? in which case there is another here.

Reply:
@GilesBathgate Thanks for your response.

Do you mean the line 250?

Reply:
> Do you mean the line 250?

Yes.